tests/page-objects/default-mobile/reports/reports.wdio.page.js

```diff
@@ -0,0 +1,90 @@
+const commonElements = require('../../default/common/common.wdio.page');
+
+const SELECT_ALL = '.mobile.multiselect-bar-container .select-all-label';
+const DESELECT_ALL = '.mobile.multiselect-bar-container .deselect-all';
+const DELETE_CONFIRM_MODAL = 'mm-modal#bulk-delete-confirm';
+const deleteAllButton = () => $('.mobile.multiselect-bar-container .bulk-delete');
+const selectedReportsCount = () => $('.mobile.multiselect-bar-container .selection-count .minimal');
+const closeOpenReportBtn = () => $('.navigation .filter-bar-back');
+
+const verifyMultiselectElementsDisplay = async (shouldHide=false) => {
+  await (await deleteAllButton()).waitForClickable( { reverse: shouldHide });
+  await (await selectedReportsCount()).waitForDisplayed({ reverse: shouldHide });
+  return {
+    countLabel: shouldHide ? false : await (await selectedReportsCount()).getText(),
+    selectedCount: (await reportsPageDefault.selectedReportsCheckboxes()).length,
+  };
+};
+
+const selectAll = async () => {
+  await (await $(SELECT_ALL)).waitForDisplayed();
+  await (await $(SELECT_ALL)).click();
+  await (await $(SELECT_ALL)).waitForDisplayed({ reverse: true });
+  return await verifyMultiselectElementsDisplay();
+};
+
+const deselectAll = async () => {
+  await (await $(DESELECT_ALL)).waitForDisplayed();
+  await (await $(DESELECT_ALL)).click();
+  await (await $(DESELECT_ALL)).waitForDisplayed({ reverse: true });
+  return await verifyMultiselectElementsDisplay(true);
+};
+
+const selectReports = async (uuids) => {
+  for (const uuid of uuids) {
+    if (!(await reportsPageDefault.isReportSelected(uuid))) {
+      await (await reportsPageDefault.reportCheckbox(uuid)).click();
+    }
+  }
+  return verifyMultiselectElementsDisplay();
+};
+
+const deselectReports = async (uuids, shouldHideElements=false) => {
+  for (const uuid of uuids) {
+    if (await reportsPageDefault.isReportSelected(uuid)) {
+      await (await reportsPageDefault.reportCheckbox(uuid)).click();
+    }
+  }
+  return verifyMultiselectElementsDisplay(shouldHideElements);
+};
+
+const deleteSelectedReports = async () => {
+  await (await deleteAllButton()).waitForDisplayed();
+  await (await deleteAllButton()).click();
+
+  await (await reportsPageDefault.bulkDeleteModal()).waitForDisplayed();
+  await (await $(`${DELETE_CONFIRM_MODAL} .btn.submit.btn-danger`)).click();
+  const confirmButton = $(`${DELETE_CONFIRM_MODAL} [test-id="bulkdelete.complete.action"]`);
+  await (await confirmButton).waitForDisplayed();
+  await (await confirmButton).click();
+  await (await reportsPageDefault.bulkDeleteModal()).waitForDisplayed({ reverse: true });
+
+  await commonElements.waitForPageLoaded();
+  await (await reportsPageDefault.reportList()).waitForDisplayed();
+};
+
+const openReport = async (reportId) => {
+  await (await reportsPageDefault.firstReport()).waitForDisplayed();
+  const reportListItem = await reportsPageDefault.reportByUUID(reportId);
```

Comment:
I guess there are better ways of extending these functions from the original page object, but I didn't have enough time to think deeper on it. Any suggestion is great help :)

Reply:
I think we can reuse openReport from the original page object (`tests/page-objects/default/reports/reports.wdio.page.js`) with just one adjustment: 
Leave `await resetFilter();` out of the function, export `resetFilter()` and use it before calling openReport on not mobile scenarios.
What do you think?

Reply:
I was checking the original `openReport()` method, and it was really weird to me why it is necessary to reset the filter before opening a report, that doesn't make any sense. I removed the [line where it calls the `resetFilter` method](https://github.com/medic/cht-core/blob/master/tests/page-objects/default/reports/reports.wdio.page.js#L305) and ran all the tests for the default config and everything passed successfully. I don't think that we need that line. 

As Lore said in the previous comment, I think that is a good idea to export the `resetFilter()` method in case anyone needs it, but for the `openReport()` method it is not necessary.

Reply:
Thank you girls! I've removed the `resetFilter()` from `openReport()` and just expose it in case anyone needs it. I ran the tests multiple times and I didn't see flakys of tests related with reports.